### DISCUSSION TITLE
Can you use this tool with languages other than English?

### DISCUSSION THREAD

**Discussion by TrueYantor** [Lailloken/Exile-UI#297] [General]
Straight forwared question, are you able to use this tool with different languages than EN?

**Comment by Lailloken** [comment]
As of right now, the only way to get 100% functionality is by playing with the client set to English. That's because certain aspects directly interact with the client, e.g. by reading item-info texts ("prefix", "suffix", "level", "strength", etc.) or the client.txt log-file ("you have entered...", "XYZ has reached level...", etc.).
Supporting more languages requires a lot of work. I'll think about implementing a feature that enables manually adding more language support, so people can create "mods/addons" and share it.
